server/borsen/io/feed_parsers/ritzau.py

```diff
@@ -41,5 +66,14 @@ def _category_filter(self, category):
                 populated_categories.append(match)
         return populated_categories
 
+    def sectionID_category_filter(self, category):
+        return self._category_filter(category, "categories", "ritzau_section_id")
```

Comment:
why is this still mapped? it seems like this should only go to sections now

Reply:
I discussed with Ivan that we need both mapping, `<section>` to `Sections CV` and `<sectionID>` to `categories CV`

Reply:
Hm, not really @devketanpro , sorry if I wasn't precise. We map the sectionID to the Sections CV, the same way we did for the Categories CV. There is no mapping to the Categories CV anymore, just adding this one category there. THanks